docs/tutorials/getting_started/moran.rst

```diff
@@ -78,14 +77,33 @@ The scores in each round::
     [7.0, 7.0, 7.0, 0.0]
     [7.0, 7.0, 7.0, 0.0]
 
+We can plot the results of a Moran process with `mp.populations_plot()`. Let's
+use a larger population to get a bit more data::
+
+   >>> import random
+   >>> import matplotlib.pyplot as plt
+   >>> axl.seed(15)  # for reproducible example
+   >>> N = 10
+   >>> players = [random.choice(axl.demo_strategies)() for i in range(N)]
```

Comment:
I think I would prefer the hard coded approach as there's less code to (potentially) understand but your call. As long as they match up.